source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift

```diff
@@ -117,6 +128,9 @@ class ACRChoiceButton: NSView, NSTextFieldDelegate, InputHandlingViewProtocol {
         button.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
         button.topAnchor.constraint(equalTo: topAnchor).isActive = true
         button.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
+        button.heightAnchor.constraint(greaterThanOrEqualToConstant: CGFloat(16)).isActive = true
+        button.widthAnchor.constraint(equalToConstant: CGFloat(16)).isActive = true
```

Comment:
Why equal to for width and >= for height?
Any reason

Reply:
Yes, we only need the width equivalence constant because it does not shrink where we set it, and height may have an impact on other elements of the stack. We require a minimum of some constant height that is not less than that. where We need the exact width, not more or less. my intentions is to not affect when there is multiline or other things in stack. let's test some edge case.